frontend/src/summary.pug

```diff
@@ -90,16 +90,14 @@
       .error-message-box__failed-repo--reason(v-else) {{ errorBlock.errorMessage }}
   .fileTypes(v-if="filterBreakdown")
     .checkboxes.mui-form--inline(v-if="Object.keys(fileTypeColors).length > 0")
-      label(
-        v-bind:style="{ 'background-color' : '#000000', 'color' : '#ffffff' }"
-      )
+      label(style='background-color: #000000; color: #ffffff')
         input.mui-checkbox--fileType(type="checkbox", v-model="checkAllFileTypes")
         span All:&nbsp;
       template(v-for="fileType in Object.keys(fileTypeColors)" )
         label(
           v-bind:key="fileType",
-          v-bind:style="{ 'background-color' : fileTypeColors[fileType], \
-            'color' : getFontColor(fileTypeColors[fileType])}"
+          v-bind:style="{ 'background-color': fileTypeColors[fileType], \
+            'color': this.getFontColor(fileTypeColors[fileType])}"
```

Comment:
please do also highlight this to be added into style check.
Is there a need to modify it in this PR?

Reply:
This PR adds several similar code block for file type lable in authorship.pug. I feel it would be better to standardize the spacing in this PR

Reply:
actually my question is, is there any difference between `getFontColor` and `this.getFontColor`? 
If there isn't then there is no need to make modifications to this file for this PR.

Reply:
I made getFontColor to be window function. Without the `this` there will be `method "getFontColor" is not defined` error and the checkboxes will not be rendered